### PR TITLE
Point CircleCI to this repo

### PR DESCRIPTION
**Pull request by gliptak**
Signed-off-by: Gábor Lipták gliptak@gmail.com

### CODE REVIEW

**Comment by gliptak** [comment]
@Sean-Holcomb please review

**Comment by AjayTripathy** [comment]
Hi @gliptak thanks so much for this PR. I think the repo we actually want to point to is quay.io/kubecost1/kubecost-cost-model:latest Mind making that change?

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,16 +7,16 @@ jobs:
       - run: | 
           TAG=0.1.$CIRCLE_BUILD_NUM
           docker login -u $DOCKER_USER -p $DOCKER_PASS
-          docker build -t ajaytripathy/kubecost-cost-model:$TAG .
-          docker push ajaytripathy/kubecost-cost-model:$TAG
+          docker build -t kubecost/cost-model:$TAG .
```
Comment (AjayTripathy) [inline .circleci/config.yml:10]: Let's use quay.io/kubecost1/kubecost-cost-model:latest for consistency with the image in the kubernetes/deployment folder

File: .circleci/config.yml [hunk 2]
```diff
@@ -7,16 +7,16 @@ jobs:
...
+          docker push kubecost/cost-model:$TAG
   deploy:
     machine: true
     steps:
       - checkout
       - run: |
           docker login -u $DOCKER_USER -p $DOCKER_PASS
-          docker build -t ajaytripathy/kubecost-cost-model:latest .
-          docker push ajaytripathy/kubecost-cost-model:latest
+          docker build -t kubecost/cost-model:latest .
```
Comment (AjayTripathy) [inline .circleci/config.yml:18]: And throughout: quay.io/kubecost1/kubecost-cost-model:latest

**Review by AjayTripathy — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (gliptak) [reply .circleci/config.yml]: updated

File: .circleci/config.yml (hunk 2 above)
Comment (gliptak) [reply .circleci/config.yml]: updated. we might need to see an actual run, I'm not sure that `docker build` allows for `quay.io` prefix

**Comment by AjayTripathy** [comment]
Hi @gliptak -- thanks so much for this! Turns out I also need to grant the circleCI job credentials to write to that repo. I will handle that and then merge.

**Comment by gliptak** [comment]
@AjayTripathy anything else on this PR?

**Comment by AjayTripathy** [comment]
Hi gliptak! thanks for following up, it is much appreciated. I still need to change the keys on our side.

**Pull request closed without merging**